meetings/2022-06-16.md

```diff
@@ -43,17 +46,38 @@
-* Sc: Implemented a view for "Storage details" - not particularly interesting/useful to the user in the context of notification stuff, but it was to show that the application is aware of what's available at this point.
+* SC: Implemented a view for "Storage details" - not particularly interesting/useful to the user in the context of notification stuff, but it was to show that the application is aware of what's available at this point.
 * SC: Simple screencast at https://github.com/solid/specification/issues/355#issuecomment-1140508784 along with other findings in the comment.
 * SC: Also implemented subscription request to `WebSocketSubscription2021` and connect to `notify:source`. I didn't yet implement an actual feature to use this - will share that in the future. Need server implementations to move forward in parallel to get anywhere with this. (Also need to have dokieli use Solid OIDC but this is a side issue.)
 * SC: Left comments/issues/PRs based on this.
-
+* eP: If GET or HEAD includes that Link relation on every resource that responds with 2xx response that seems to solve the problem.
+* SC: May not be limited to 2xx.
 
 ### Discovery of Notification Subscription Metadata
 URL: https://github.com/solid/notifications/issues/58
 
 * SC: I'll summarise https://github.com/solid/notifications/issues/58#issuecomment-1151117524 .
-
+* CB: Each resource points to a notification discovery resource or if every resource in a container is managed by the same channel. If I subscribe to a container, I get a notification about the container. Is that intended to be the same or buggy?
+* CB: question was asked if every resource in the storage points to the same description resource with information about subscribing to notification channels
+* SC: Generally when the resource description changes. The assumption is that you get notification, so if the containement information changes you would get a notification.
+* SC: Forget about the prior/old/insecure websocket behaviour that's not the one described here.
+* eP: We may need to document use cases to subscribe to contained resources
+* RG: How much of this should be on the server and how much on the client? Have we thought about how to do this in terms of implementation efficiency and balancing that against with the ease of making a client.
+* SC: Not that I know of anything that's documented. Which is why we are hoping for implementation feedback to approach those questions.
+* ...: Myself I understood the protocol better after implementing some of it, and what things needs to be clarified in the spec or missing.
+* CB: Rahul has a point, the client side should always work, however if you imagine a web app that accesses a lot of resources on the server, it may affect scalability to have separate websockets connections.
```

Comment:
```suggestion
* CB: Rahul has a point, the client side should always work. However, if you imagine a web app that accesses a lot of resources on the server, it may affect scalability to have separate websocket connections.
```